Documentation/General/productVision.tex

```diff
@@ -0,0 +1,70 @@
+\documentclass[10pt,a4paper]{article}
+\usepackage[utf8]{inputenc}
+\usepackage{amsmath}
+\usepackage{amsfonts}
+\usepackage{amssymb}
+\begin{document}
+
+\title{Product Vision\\
+	\begin{small}
+	The why, who and what
```

Comment:
I would rather go for:
```
\title{MoodCat\\
	\begin{small}
	Product Vision
```

`The why, who and what` might be whats in there, but it's not a catchy title.